src/main/java/uk/gov/ons/ctp/response/action/export/service/ManifestBuilder.java

```diff
@@ -0,0 +1,55 @@
+package uk.gov.ons.ctp.response.action.export.service;
+
+import com.fasterxml.jackson.databind.ObjectMapper;
+import java.io.ByteArrayOutputStream;
+import java.io.IOException;
+import java.time.Instant;
+import java.time.format.DateTimeFormatter;
+import java.util.ArrayList;
+import java.util.Arrays;
+import java.util.List;
+import org.apache.commons.codec.digest.DigestUtils;
+import org.springframework.stereotype.Component;
+import uk.gov.ons.ctp.response.action.export.domain.PrintFileMainfest;
+import uk.gov.ons.ctp.response.action.export.domain.PrintFilesInfo;
+
+@Component
+public class ManifestBuilder {
+  private final ObjectMapper objectMapper;
+
+  public ManifestBuilder(ObjectMapper objectMapper) {
+    this.objectMapper = objectMapper;
+  }
+
+  public ByteArrayOutputStream createManifestData(String filename, ByteArrayOutputStream data)
+      throws IOException {
+    PrintFileMainfest printFileMainfest = createManifest(filename, data);
+
+    String jsonManifest = objectMapper.writeValueAsString(printFileMainfest);
+    byte[] manifestBytes = jsonManifest.getBytes();
+    ByteArrayOutputStream byteArrayOutputStream = new ByteArrayOutputStream(manifestBytes.length);
+    byteArrayOutputStream.write(manifestBytes);
+
+    return byteArrayOutputStream;
+  }
+
+  private PrintFileMainfest createManifest(String filename, ByteArrayOutputStream data) {
+    String checksum = DigestUtils.md5Hex(data.toByteArray());
+
+    PrintFilesInfo printFilesInfo = new PrintFilesInfo(data.size(), checksum, ".\\", filename);
+    List<PrintFilesInfo> files = new ArrayList<>(Arrays.asList(printFilesInfo));
+
+    String manifestCreatedDateTime = DateTimeFormatter.ISO_INSTANT.format(Instant.now());
+
+    // These are hardcoded for now, they will be update and passed in from the actionsvc in the
+    // future
+    return new PrintFileMainfest(
```

Comment:
Spelling error in manifest. Should this be hardcoded or take values from templatemapping table. I think the changes to Action Service are a while away yet.

Reply:
@LukeLoze1 this is to stay hard-coded for now, isn't it?

Reply:
OK, so we will resolve this in RMC-119 https://trello.com/c/oRPSTwHn/594-rmc-119-action-exporter-service-to-generate-initial-contact-letter-print-file-wales-5